.gitmodules

```diff
@@ -10,6 +10,7 @@
 [submodule "src/external/glbinding"]
 	path = src/external/glbinding
 	url = https://github.com/cginternals/glbinding.git
+	branch = v3.0
```

Comment:
Is this change accident? You haven't mentioned this change in the description

Reply:
I have mentionned it on description, and no, it juste add what branch master already used

Reply:
I'm not sure if moving away from master branch is a great idea. May need others to comment on this

Reply:
What ? I move nothing, i juste add branch info mission

Reply:
Im sure that those are not default branch ;/

Reply:
No, i have juste added branch corresponding what submodule used